### PR TITLE
Feature/86cv684zq messenger inbox component

### PR DESCRIPTION
**Pull request by viggy21**
Added the messenger inbox feature to the left side of the messages page.

### CODE REVIEW

File: behind-the-veil-siteroot/imports/ui/components/pages/messages/MessagesPage.jsx [hunk 1]
```diff
@@ -7,18 +7,41 @@
...
  * Messages page with all users they've chatted with
  */
+
+// Dummy data
+const messagePreviews = [
+    {
+        userName: 'titan1',
+        name: 'Annie',
+        recentMessage: 'Hi, I was wondering if you were the amazing artist that he recommended?',
+        image: '',
```
Comment (nethbtju) [inline behind-the-veil-siteroot/imports/ui/components/pages/messages/MessagesPage.jsx:22]: From the way the database schema is currently written you will be getting a photoid with the message object which you might have to fetch from the photos db: https://lucid.app/lucidchart/bf2238cb-8db1-4a45-a72d-b6007c65c46e/edit?viewport_loc=-3967%2C-1545%2C10533%2C3907%2C0_0&invitationId=inv_99b49be3-2c60-4b42-a5c7-16051f9d3b7e

Comment (nethbtju) [reply behind-the-veil-siteroot/imports/ui/components/pages/messages/MessagesPage.jsx:22]: Let us know if you have a better way you'd like to do it

File: behind-the-veil-siteroot/imports/ui/components/pages/messages/MessagesPage.jsx
Comment (nethbtju) [inline behind-the-veil-siteroot/imports/ui/components/pages/messages/MessagesPage.jsx:1]: Add you name as a contributor

Comment (nethbtju) [inline behind-the-veil-siteroot/imports/ui/components/pages/messages/MessagesPage.jsx:1]: Try viewing this in mobile view, it throws everything out of a bit whack haha see if you can get it to scale accordingly

File: behind-the-veil-siteroot/imports/ui/components/whiteBackground/whiteBackground.css [hunk 2]
```diff
@@ -30,7 +30,7 @@
 
 .message-left {
     @apply rounded-b-none rounded-r-none; /*dont round bottom and left corners for this*/
-    @apply mt-16 p-10 w-3/12 h-fit; /*some positioning variables*/
+    @apply mt-16 p-4 w-3/12 h-fit; /*some positioning variables*/
```
Comment (nethbtju) [inline behind-the-veil-siteroot/imports/ui/components/whiteBackground/whiteBackground.css:33]: what did this end up doing btw? js curious!

**Review by nethbtju — CHANGES_REQUESTED** [review]
couple of changes needed

**Review by nethbtju — CHANGES_REQUESTED** [review]
Changes needed to approve

**Comment by NikkiShao** [comment]
A new branch was made for messenger inbox, this is now not relevant anymore

**Pull request closed without merging**